Teapot.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we .gitignore this new file?

Reply:
no, since it's in `xcshareddata`

Reply:
do you know what it's for?

Reply:
Per the 9.3 release notes, storing various checks that only need to be done once: https://stackoverflow.com/a/49564624/681493